src/containers/Auth/ConfirmOTP/ConfirmOTP.tsx

```diff
@@ -105,6 +105,7 @@ export const ConfirmOTP = () => {
           otp: values.OTP,
           email: userObject.email,
           consent_for_updates: userObject.consent_for_updates,
+          organization_name: userObject.organization_name,
         },
```

Comment:
https://github.com/glific/glific/pull/4615/files#r2576381124
We dont need an additional field for org name